include/spm/evt_mario.h

```diff
@@ -135,6 +141,5 @@ UNKNOWN_FUNCTION(func_800f3310);
 UNKNOWN_FUNCTION(func_800f3334);
 UNKNOWN_FUNCTION(evt_mario_check_3d);
 UNKNOWN_FUNCTION(func_800f33b0);
-UNKNOWN_FUNCTION(evt_mario_calc_damage_to_enemy);
```

Comment:
This just seems to have been removed entirely for no reason?

Reply:
idk why I did that sorry lol